### PR TITLE
Http URL mapping enhancement

### PR DESCRIPTION
**Pull request by pavolloffay**
@objectiser could you please review?

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay looks fine, although might be worth providing some unit tests just to check the behaviour - also with just a path specified, to make sure URL gets correctly setup.

**Comment by pavolloffay** [comment]
@objectiser unit tests added

File: server/api/src/test/java/org/hawkular/apm/server/api/model/zipkin/SpanTest.java [hunk 1]
```diff
@@ -71,12 +71,36 @@ public void testPartialUrl() throws MalformedURLException {
...
+
+        Span span = new Span(Arrays.asList(ba));
+
+        URL spanURL = span.url();
+        assertEquals(url, spanURL);
+    }
+
+    @Test
+    public void testUrlFromPath() throws MalformedURLException {
+        URL url = new URL("http://localhost:8080/my/path");
```
Comment (objectiser) [inline server/api/src/test/java/org/hawkular/apm/server/api/model/zipkin/SpanTest.java]: Could you change this to be just the path, e.g. "/my/path" - to ensure it is still returned as a URL (even though missing the host and port).

Comment (pavolloffay) [inline server/api/src/test/java/org/hawkular/apm/server/api/model/zipkin/SpanTest.java]: done

**Comment by objectiser** [comment]
Thanks Pavol

**Pull request merged**